docs/docs/routing.md

```diff
@@ -0,0 +1,238 @@
+---
+id: routing
+title: Routing
+permalink: docs/routing.html
+---
+
+Routing is the process of choosing what to render based on the browser's current URL.
+
```

Comment:
We also need to make it clear that React doesn't *require* you to implement routing. For example, if you *don't* write a single page app, you can use React without these techniques.